Cargo.toml

```diff
@@ -5,3 +5,6 @@ members = [
     "conjure-verification",
     "conjure-verification-api",
 ]
+
+[patch.crates-io]
```

Comment:
We're depending on some hyper changes that haven't released yet, but I expect that will be resolved by the time we finish porting everything.